apps/element-storybook/src/migration.mdx

```diff
@@ -81,16 +81,17 @@ The migration theme is accessible by passing `theme='legacyBS'` to the `ThemePro
 #### Recommended Early Component Swaps
 
 - layout (`Box`, `Grid`, `Container`)
-- button?
 - table
 - menu
 - list (excluding list-group)
+- alert
+- card
 
 ### Form
 
 Where you're coming from will determine your strategy for form migration. We have moved away from `formik` (used by `@availity/form`) in favor of `react-hook-forms`. We have also provided the base components to utilize with your own logic/libraries.
```

Comment:
Small typo with `react-hook-forms`. There is an added `s` at the end

Reply:
good catch. fixed